SPECS/glusterfs/glusterfs.spec

```diff
@@ -1189,23 +1607,23 @@ exit 0
 %ghost %dir %attr(0755,-,-) %{_sharedstatedir}/glusterd/vols
 
 # Extra utility script
-%dir %{_datadir}/glusterfs
+%dir %{_libexecdir}/glusterfs
 %dir %{_datadir}/glusterfs/scripts
-%{_datadir}/glusterfs/scripts/stop-all-gluster-processes.sh
-%{_libexecdir}/glusterfs/mount-shared-storage.sh
-%{_datadir}/glusterfs/scripts/control-cpu-load.sh
-%{_datadir}/glusterfs/scripts/control-mem.sh
+     %{_datadir}/glusterfs/scripts/stop-all-gluster-processes.sh
+%if ( 0%{?_with_systemd:1} )
+     %{_libexecdir}/glusterfs/mount-shared-storage.sh
+     %{_datadir}/glusterfs/scripts/control-cpu-load.sh
+     %{_datadir}/glusterfs/scripts/control-mem.sh
+%endif
 
 # Incrementalapi
-%dir %{_libexecdir}/glusterfs
-%{_libexecdir}/glusterfs/glusterfind
-%{_libexecdir}/glusterfs/peer_add_secret_pub
+     %{_libexecdir}/glusterfs/glusterfind
 %{_bindir}/glusterfind
+     %{_libexecdir}/glusterfs/peer_add_secret_pub
 
 %if ( 0%{?_with_firewalld:1} )
-%{_libdir}/firewalld/services/glusterfs.xml
+%{_prefix}/lib/firewalld/services/glusterfs.xml
```

Comment:
Previous macro was more accurate/less error-prone in case we ever touch these macros.

I think it might be worth to run the spec linter for this spec once other changes are done to update it to use our path macros.

Reply:
modified per suggestion